py/selenium/webdriver/common/bidi/emulation.py

```diff
@@ -399,3 +399,35 @@ def set_screen_orientation_override(
             params["userContexts"] = user_contexts
 
         self.conn.execute(command_builder("emulation.setScreenOrientationOverride", params))
+
+    def set_user_agent_override(
+        self,
+        user_agent: Optional[str] = None,
+        contexts: Optional[list[str]] = None,
+        user_contexts: Optional[list[str]] = None,
```

Comment:
Since we are now requiring Python 3.10+, we can use the newer union type syntax for type hints. I have #16590 open that converts all of our type hints.

Instead of using `Optional`, I think these should be written as:

```
user_agent: str | None = None,
contexts: list[str] | None = None,
user_contexts: list[str] | None = None,
```